ui/src/views/settings/SettingsQueue.vue

```diff
@@ -0,0 +1,321 @@
+        this.error.maximal_queue_lifetime = this.$t("common.required");
+
+        if (isValidationOk) {
+          this.focusElement("maximal_queue_lifetime");
+          isValidationOk = false;
+        }
+      }
+      return isValidationOk;
+    },
+    async setQueueSettings() {
+      if (!this.validateSetQueueSettings()) {
+        return;
+      }
+      this.loading.setQueueSettings = true;
+      this.error.setQueueSettings = "";
+      this.error.maximal_queue_lifetime = "";
+      const taskAction = "set-queue-settings";
+      const eventId = this.getUuid();
+
+      // register to task error
+      this.core.$root.$once(
+        `${taskAction}-aborted-${eventId}`,
+        this.setQueueSettingsAborted
+      );
+
+      // register to task validation
+      this.core.$root.$once(
+        `${taskAction}-validation-failed-${eventId}`,
+        this.setQueueSettingsValidationFailed
+      );
+
+      // register to task completion
+      this.core.$root.$once(
+        `${taskAction}-completed-${eventId}`,
+        this.setQueueSettingsCompleted
+      );
+
+      const res = await to(
+        this.createModuleTaskForApp(this.instanceName, {
+          action: taskAction,
+          data: {
+            maximal_queue_lifetime: parseInt(this.maximal_queue_lifetime),
+          },
+          extra: {
+            title: this.$t("action." + taskAction),
+            description: this.$t("common.processing"),
+            eventId,
+          },
+        })
+      );
+      const err = res[0];
+
+      if (err) {
+        console.error(`error creating task ${taskAction}`, err);
+        this.error.setQueueSettings = this.getErrorMessage(err);
+        this.loading.setQueueSettings = false;
+        return;
+      }
+    },
+    setQueueSettingsAborted(taskResult, taskContext) {
+      console.error(`${taskContext.action} aborted`, taskResult);
+      this.error.setQueueSettings = this.$t("error.generic_error");
+      this.loading.setQueueSettings = false;
+    },
+    setQueueSettingsValidationFailed(validationErrors) {
+      this.loading.setQueueSettings = false;
+      let focusAlreadySet = false;
+
+      for (const validationError of validationErrors) {
+        const param = validationError.parameter;
+
+        // set i18n error message
+        this.error[param] = this.getI18nStringWithFallback(
+          "settings_mailboxes." + validationError.error,
+          "error." + validationError.error,
+          this.core
+        );
+
+        if (!focusAlreadySet && param != "(root)") {
+          this.focusElement(param);
+          focusAlreadySet = true;
+        }
+      }
+    },
+    setQueueSettingsCompleted() {
+      this.loading.setQueueSettings = false;
+
+      // reload settings
+      this.getQueueSettings();
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+@import "../../styles/carbon-utils";
+
+.cv-form .bx--form-item {
+  margin-bottom: $spacing-07;
+}
+
+.toggle-dependent {
```

Comment:
This CSS class is not used and can be removed

Reply:
done